packages/react/src/components/ListBox/ListBoxMenu.js

```diff
@@ -23,6 +23,7 @@ const ListBoxMenu = ({ children, id, ...rest }) => (
     id={`${id}__menu`}
     className={`${prefix}--list-box__menu`}
     role="listbox"
+    aria-multiselectable="true"
```

Comment:
Would this be specified in `MultiSelect` JS on `ListBox.Menu` since `ListBoxMenu` is shared between the different dropdown variants?

Reply:
Fixed